wlip-0003/preliminary-word-lists/english_us

```diff
@@ -1180,6 +1199,16 @@ steward
 translator
 writer
 
+# group: religion
```

Comment:
Although I find the 'religion' word fine for this list, I wouldn't put any religion-related words here - I just updated Contributors FAQ. Please remove this group.